tortoise/contrib/test/__init__.py

```diff
@@ -258,10 +261,28 @@ def requireCapability(connection_name: str = 'models', **conditions: Any):
     def decorator(test_item):
         @wraps(test_item)
         def skip_wrapper(*args, **kwargs):
-            db = Tortoise.get_connection(connection_name)
-            for key, val in conditions.items():
-                if getattr(db.capabilities, key) != val:
-                    raise SkipTest('Capability {key} != {val}'.format(key=key, val=val))
+            checkCapability(connection_name, **conditions)
             return test_item(*args, **kwargs)
         return skip_wrapper
     return decorator
+
+
+def checkCapability(connection_name: str = 'models', **conditions: Any):
```

Comment:
The terminology is unexpected.
What this does is skip the test in the middle of it, if the capability is not set up.
And the only use-case is for testing capabilities when testing schema generation.
Makes me think this is not really necessary for being part of the API?

Reply:
So, should I instead get the connection, check for capabilities and skip the test (essentially copying the behavior of `requireCapability` but not as a decorator) in these specific tests?

Reply:
I am in two minds about it. Primarily I'm unhappy with the naming, then I thought about if we really want to expose a piece of API that we probably don't need to expose.
So I'd say copy the code into the test module.

Reply:
I thought Travis was set up that way... Let me check.
It is like that. It only auto cancels a job that haven’t been started unfortunately.